specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -248,6 +258,44 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "semanticErrorHandling",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "fail",
+              "partial"
+            ],
+            "x-ms-enum": {
+              "name": "SemanticErrorHandling",
+              "modelAsString": false,
+              "values": [
+                {
+                  "value": "fail",
+                  "name": "Fail",
+                  "description": "Default behavior. If there is an exception during semantic processing, the query will fail and return the appropriate HTTP code depending on the error."
+                },
+                {
+                  "value": "partial",
+                  "name": "Partial",
+                  "description": "If there is an exception during semantic processing, the query return partial results with a 206 status code. Partial result content depends on which semantic step failed and the reason for the failure."
+                }
+              ]
+            },
+            "description": "A value that specifies which response to return when semantic processing fails.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "semanticMaxWaitInMilliseconds",
+            "in": "query",
+            "type": "integer",
```

Comment:
This should have a property `"format"` indicating whether this is an `int32` or `int64` integer type.

Reply:
The other warnings raised by the `LintDiff` check should also be looked into

Reply:
The other warnings raised by `LintDiff` relate to missing `type:object` specifications. Our swagger (with three exceptions) has a history of omitting this, so I'll leave those be. I fixed the `int32` property.

Reply:
Please either fix it now or open (and link) to a tracking issue to clean it up.  It's technical debt that leads to the same round of questions in future revisions.

Reply:
All warnings related to `LintDiff` should be fixed now, though it seems like the CI is blocked by a merge conflict in `custom-words.txt`